DataFormats/SiStripCluster/interface/SiStripApproximateClusterv1.h

```diff
@@ -0,0 +1,31 @@
+#ifndef DATAFORMATS_SISTRIPAPPROXIMATECLUSTERv1_H
+#define DATAFORMATS_SISTRIPAPPROXIMATECLUSTERv1_H
+
+#include <vector>
+#include <numeric>
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+class SiStripApproximateClusterv1  {
```

Comment:
do you really need to have the versioning in the class name ?

Reply:
Just to understand, so you foresee to have different reduced format implemenations at the same time to co-exist in cmssw?

Reply:
At least in the initial stages yes. We need to validate compression and physics performance on a big sample before selecting the ideal format. It could be done incrementally but not sure being optimal.

Reply:
Thanks for the clarification. I also don't see a clean way around...

Reply:
ok, then I resolve this conversation but please let me know if something else comes to your mind